src/main/kotlin/com/demonwav/mcdev/i18n/inspections/TranslationInspection.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2017 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.i18n.inspections
+
+import com.demonwav.mcdev.platform.mcp.McpModuleType
+import com.intellij.codeInspection.BaseJavaLocalInspectionTool
+import com.intellij.codeInspection.InspectionManager
+import com.intellij.codeInspection.ProblemDescriptor
+import com.intellij.codeInspection.ProblemsHolder
+import com.intellij.psi.PsiElementVisitor
+import com.intellij.psi.PsiFile
+
+abstract class TranslationInspection : BaseJavaLocalInspectionTool() {
+    protected abstract fun buildVisitor(holder: ProblemsHolder): PsiElementVisitor
+
+    override final fun buildVisitor(holder: ProblemsHolder, isOnTheFly: Boolean): PsiElementVisitor {
```

Comment:
We haven't switched over yet to the Jetbrains Kotlin style, but we will soon (at least to a slightly modified version). So, in that case, the modifier order is incorrect (`final` comes before `override`).